src/Humanizer.Tests/InflectorTests.cs

```diff
@@ -21,45 +21,47 @@
 //IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 //CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-using System.Collections;
 using System.Collections.Generic;
+using Humanizer.Inflections;
 using Xunit;
 using Xunit.Extensions;
 
 namespace Humanizer.Tests
 {
-    public class InflectorTests 
+    public class InflectorTests
     {
         public readonly IList<object[]> PluralTestData = new List<object[]>();
 
+        // All singular/pluralize tests just need to demonstrate that they're 
+        // delegating to the default vocab.
         [Theory]
-        [ClassData(typeof(PluralTestSource))]
+        [InlineData("mouse", "mice")]
         public void Pluralize(string singular, string plural)
         {
-            Assert.Equal(plural, singular.Pluralize());
+            Assert.Equal(plural, Vocabularies.Default.Pluralize(singular));
```

Comment:
> ..., but testing for equality here is pretty close.

Not really, the test now verifies only that `Vocabularies.Default` can process **mouse/mice**. It's relation to `Inflector` is gone, as you changed the test. You can change the implementation of the `Inflector` to anything and this test wouldn't break.